GameV0Component.js

```diff
@@ -23,20 +25,34 @@ class GameV0Component extends EntityComponent {
   }
 
   think(dt) {
+    this.next_spawn_time -= dt;
```

Comment:
variable is uninitialized

member variables should be lowerCamelCase and prefixed with underscores if private